enaml/qt/qt_mpl_canvas.py

```diff
@@ -10,7 +10,10 @@
 from enaml.widgets.mpl_canvas import ProxyMPLCanvas
 
 from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg
-from matplotlib.backends.backend_qt4agg import NavigationToolbar2QTAgg
+try:
+    from matplotlib.backends.backend_qt4agg import NavigationToolbar2QTAgg
+except ImportError:
+    from matplotlib.backends.backend_qt4agg import NavigationToolbar2QT
```

Comment:
-1. 

I think those imports are there explicitly because they are Qt4 compatible. Using a Qt5 import would be confusing, since we don't support Qt5.